src/app/shared/table/totals-table.component.ts

```diff
@@ -0,0 +1,61 @@
+import { Component, Input, Output, EventEmitter } from '@angular/core';
+import { TotalsTableRow, RouterParams, getGroupName, CHARTTYPE_HORIZBAR } from '../../ngrx/';
+import { neutralColor, standardColor } from '../util/chart.util';
+
+@Component({
+  selector: 'metrics-totals-table',
+  template: `
+    <div class="table">
+      <div class="header row">
+        <div>{{getGroupName(routerParams.metricsType, routerParams.group)}}</div>
+        <div class="number">Downloads</div>
+        <div class="number growth"></div>
+      </div>
+      <div class="row" *ngFor="let data of tableData.slice(0, numRowsWithToggle)">
+        <div>
+          <prx-checkbox small [checked]="data.charted" [color]="routerParams.chartType === horizbar ? blue : data.color"
+                        (change)="toggleEntry.emit({podcastId: routerParams.podcastId, groupName: data.label, charted: $event})">
+            {{data.label}}
+          </prx-checkbox>
+        </div>
+        <div class="number">{{data.value | largeNumber}} <span class="percent">({{data.percent.toPrecision(2)}}%)</span></div>
+        <div class="number growth"></div>
+      </div>
+    </div>
+
+    <div class="table">
+      <div *ngIf="tableData?.length > numRowsWithToggle && numRowsWithToggle > 0" class="other header row">
```

Comment:
![image](https://user-images.githubusercontent.com/1410587/44875536-b8f70d00-ac5b-11e8-9510-cb8ef6f73d73.png)

I find the start of the "Others:" section a bit weird.  Wish there was some other kind of separator, maybe even an `<hr>` or something.  The white box makes the previous data row look related to "Others" somehow.  But I can be convinced to live with this for now!